bigframes/ml/llm.py

```diff
@@ -132,6 +135,9 @@ def __init__(
         session: Optional[bigframes.Session] = None,
         connection_name: Optional[str] = None,
     ):
+        if model_name == _TEXT_EMBEDDING_004_ENDPOINT:
+            msg = exceptions.format_message(_REMOVE_DEFAULT_MODEL_WARNING)
+            warnings.warn(msg, category=FutureWarning, stacklevel=2)
```

Comment:
No. If they explicitly set this, we shouldn't show a warning.

Try something like this instead:

```suggestion
        model_name: Optional[Literal[
            "text-embedding-005",
            "text-embedding-004",
            "text-multilingual-embedding-002",
        ]] = None,
        session: Optional[bigframes.Session] = None,
        connection_name: Optional[str] = None,
    ):
        if model_name is None:
            model_name = _TEXT_EMBEDDING_004_ENDPOINT:
            msg = exceptions.format_message(_REMOVE_DEFAULT_MODEL_WARNING)
            warnings.warn(msg, category=FutureWarning, stacklevel=2)
```

Reply:
Please apply the same for the other models, too.

Reply:
Thanks Tim, I used to believe we plan to remove default model for Bigframes 3.0, and throw an warning for default model removal in Bigframes 2.0. If I apply the same change, we are removing default model in bigframes 2.0. Please advise should I remove default models in Bigframes 2.0, or should we wait until 3.0 release time? Thanks a lot.